tv_apps/smart-system/locales/system.en-US.properties

```diff
@@ -362,3 +362,6 @@ dismiss-pairing-code=Dismiss
 pin-to-home=Pin to home
 fte-pin-to-home-description=Pin TV channels, application-devices and browser pages to Home as a shortcut.
 
+# Preview App
+preview-app-hint-title=Press OPTION to
+preview-app-hint-text=Add to Apps
```

Comment:
We shouldn't split sting in this way, it's hard to localized to other languages. Put the whole string in text, and leave title blank or use `Tips`. Need info UX on the bug to fix title.